app/assets/javascripts/admin/addon/components/admin-plugin-config-top-nav.gjs

```diff
@@ -16,9 +16,9 @@ export default class AdminPluginConfigTopNav extends Component {
   }
 
   <template>
-    <div class="admin-controls">
+    <div class="admin-nav-submenu">
       <HorizontalOverflowNav
-        class="nav-pills action-list main-nav nav plugin-nav"
+        class="plugin-nav admin-plugin-config-page__top-nav"
```

Comment:
Does including the page name in the class name mean we'll need to duplicate the CSS for each page?

Reply:
Yeah I think maybe `admin-plugin-config-page` will change at some point, we might want a more generic way of representing this, but for now I think it's OK since it's only affecting these plugin pages.